pkg/project/admission/lifecycle/admission.go

```diff
@@ -60,6 +60,11 @@ func (e *lifecycle) Admit(a admission.Attributes) (err error) {
 	if len(a.GetNamespace()) == 0 {
 		return nil
 	}
+	// always allow a SAR request through, the SAR will return information about
+	// the ability to take action on the object, no need to verify it here.
+	if isSubjectAccessReview(a) {
```

Comment:
Can we remove the subject access review related resources from line 55:

```
 var recommendedCreatableResources = sets.NewString("subjectaccessreviews", "resourceaccessreviews", "localsubjectaccessreviews", "localresourceaccessreviews")
```

Seems unnecessary now that we special case it much further up in the block.